worker/charmrevision/doc.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package charmrevision defines the charm revision updater worker. This worker
+// is responsible for polling Charmhub every 24 hours to check if there are new
+// revisions available of any charm deployed in the model. If so, it will put a
+// document in the Juju database, so that the next time the user runs
```

Comment:
```suggestion
// revisions available of any repository charm deployed in the model. If so, it
// will put a document in the Juju database, so that the next time the user runs
```